lib/hammer_cli_katello/content_export_helper.rb

```diff
@@ -1,12 +1,18 @@
 module HammerCLIKatello
   module ContentExportHelper
     include ApipieHelper
+
     def execute
       response = super
-      if option_async? || response != HammerCLI::EX_OK
+      if option_async?
+        output.print_message _("Once the task completes the export metadata must be generated "\
+          + "with the command: hammer content-export generate-metadata "\
```

Comment:
Can ya add quotes to the command or a new line. Like

```suggestion
          + "with the command:\n hammer content-export generate-metadata "\
```

Reply:
or even
```ruby
        output.print_message _("Once the task completes the export metadata must be generated "\
          + "with the command:")
        output.print_message("hammer content-export generate-metadata --task-id #{@task['id']}")
```
I dont think the command itself needs translation right?